### DISCUSSION TITLE
Optolink Bezugsquelle

### DISCUSSION THREAD

**Discussion by githubhuhn** [openv/openv#612] [General]
Hallo
Ich bin auch an so einem Projekt interessiert. Ist aus der Serie etwas geworden und gibt es noch eine Schnittstelle zu kaufen?
Habe bereits nach kommerziellen Teilen gesucht. ebay hat einige Treffer unter dem Stichwort Optolink, aber das hat nicht das V sondern sieht komplett anders aus. Nach was muss ich suchen? Es gibt seltsamerweise nichts außer Selbstbauprojekten.
Grüße!

**Comment by speters** [comment]
Hast du schonmal wg. der [kleinen USB Optolinkplatine](https://github.com/openv/openv/wiki/Bauanleitung-USB#bauanleitung-f%C3%BCr-schaltung-nr-4-neue-variante-mit-kleineren-abmessungen-micro-usb) angefragt? In dem Wiki-Artikel steht eine Mailadresse, vielleicht kann da auch @STB3 (als Verfasser der Wiki-Seite) was sagen.

Das originale USB-Kabel (Viessmann Optolink  7856059) ist leider teuer und schwer zu bekommen.
Bei _kleinanzeigen.de_ kann man Glück haben - da sind z.B. gerade eben zwei Angebote um 40€ drin.

**Comment by STB3** [comment]
Hi!
Ja, ich habe noch welche, einfach an die Mailadresse schreiben.
Grüße
Dirk

**Comment by githubhuhn** [comment]
Hallo, ich habe tatsächlich noch eine offizielle Schnittstelle bekommen und habe das System schon grundlegend am laufen. Zumindest so dass ich über telnet Anfragen starten kann.

Was mir allerdings fehlt ist eine vito.xml Datei für mein Gerät. Ein paar Abfragen mit der vito.xml eines anderen Typs funktionieren auch bei mir, andere nicht. Ich habe eine Vitocal-A mit der Vitovent dran hängend. Gerätetyp ist 204C. Man findet für einige Typen was, aber für diesen leider nicht. Ist das so ein Exot?

Habt ihr Tips wie ich vorgehen könnte?

**Reply by philippoo66** [reply]
![grafik](https://github.com/openv/openv/assets/122479122/0482b76f-674e-400d-a13d-e3a4ed77dbd8)

[DP_CU401B_A.txt](https://github.com/openv/openv/files/15060488/DP_CU401B_A.txt)

es ist immer wieder das Gleiche und ganz einfach und hier beschrieben https://github.com/philippoo66/ViessData21?tab=readme-ov-file#dp_listen_2zip

**Reply by githubhuhn** [reply]
Diese Datei kenne ich schon, da ist aber etliches drin was nicht funktioniert oder wofür ich in anderen leider sehr unvollständigen Quellen andere Adresswerte gefunden habe.
Beispiel Außentemperatur. In der Datei steht die als 0x01C1, aber wirklich ist es 0x0101.
Was ist da anders als ich es mir vorstelle?

**Reply by philippoo66** [reply]
versuch mal die hier, da steht die AT zumin mit 0101h drin
 
[Datapoints.VBC700_BW_WW.2046.txt](https://github.com/openv/openv/files/15454559/Datapoints.VBC700_BW_WW.2046.txt)

scheinbar gibt es unterschiedliche Datensätze bei unterschiedlichen Geräten...

**Reply by philippoo66** [reply]
kennst du schon mein neuestes Projekt zu dem Thema, den [Optolink Splitter](https://github.com/philippoo66/optolink-splitter)? der macht vieles einfacher ('Installation' und auch Stichwort xmls - die braucht es dabei nicht), besonders auch Haus Automatisierung/MQTT. Auch wenn man garnicht splitten will...

**Reply by githubhuhn** [reply]
Vielen Dank, schau ich mir beides an :-)
